lib/helpers/environment.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Helper functions pretaining to the environment domain
+*/
+
+use Automattic\VIP\Environment;
+
+/**
+ * A wrapper for Automattic\VIP\Environment::get_var
```

Comment:
PHPDoc is missing the signature for arguments, let's add that.

Reply:
Good catch. I have made this update in the latest commit.